source/ui/pixel-display-painted-item.cpp

```diff
@@ -0,0 +1,253 @@
+/*
+* C++ Assembler Interpreter
+* Copyright (C) 2016 Chair of Computer Architecture
+* at Technical University of Munich
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "ui/pixel-display-painted-item.hpp"
+
+#include <algorithm>
+#include <ctime>
+#include <iostream>
+
+#include "common/assert.hpp"
+#include "ui/output-component.hpp"
+
+PixelDisplayPaintedItem::PixelDisplayPaintedItem(QQuickItem *parent)
+: QQuickPaintedItem(parent), _options{} {
+  _options.colorMode = 0;
```

Comment:
Why don't you give your options a proper constructor for this? Then you can initialize it in the initializer list